apps/filecoin-site/src/app/_components/Footer/DesktopFooter.tsx

```diff
@@ -1,16 +1,13 @@
-import clsx from 'clsx'
-
 import { NewsletterForm } from '@/components/Footer/NewsletterForm'
 import { Logo } from '@/components/Logo/Logo'
 
-import { CONTAINER_SPACING } from './Footer'
 import { LegalSection } from './LegalSection/LegalSection'
 import { Navigation } from './Navigation'
 
 export function DesktopFooter() {
   return (
     <div className="hidden md:flex md:flex-col md:gap-16 md:py-20">
-      <div className={clsx('flex justify-between gap-8', CONTAINER_SPACING)}>
+      <div className="flex justify-between gap-8">
```

Comment:
@CharlyMartin - The `CONTAINER_SPACING` serves a purpose. Please have a look at [this message](https://filecoinfoundation.slack.com/archives/C08P3085Y3E/p1751539081433819).